src/ducks/Signals/utils.js

```diff
@@ -22,13 +22,37 @@ const getTarget = target => {
   return { value: slug, label: slug }
 }
 
+export const DAILY_ACTIVE_ADDRESSES = 'daily_active_addresses'
+export const PRICE_PERCENT_CHANGE = 'price_percent_change'
+export const PRICE_VOLUME_DIFFERENCE = 'price_volume_difference'
+
+export const PRICE_CHANGE_TYPES = {
+  MOVING_UP: 'percent_up',
+  MOVING_DOWN: 'percent_down'
+}
+export const pricePercentChangeUp = {
+  value: PRICE_PERCENT_CHANGE,
```

Comment:
Why `pricePercentChangeUp` should store a pointer to the `PRICE_CHANGE_TYPES` data structure?